pasta_eln/GUI/project.py

```diff
@@ -314,6 +314,7 @@ def modelChanged(self, item:QStandardItem) -> None:
     Args:
       item (QStandardItem): item changed, new location
     """
+    print('start project model changed')
```

Comment:
Please use log.info instead of print, if not needed please remove the comment

Reply:
@jmurugan-fzj  Thanks, this was an old debugging message, I forgot to remove. Removed it now